core/benches/blocks/common.rs

```diff
@@ -182,9 +182,12 @@ pub fn build_state(rt: &tokio::runtime::Handle, account_id: &AccountId) -> State
     {
         let mut state_block = state.block();
 
-        state_block.config.transaction_limits = TransactionLimits::new(u64::MAX, u64::MAX);
-        state_block.config.executor_runtime.fuel_limit = u64::MAX;
-        state_block.config.executor_runtime.max_memory = u32::MAX.into();
+        state_block.world.parameters.transaction = TransactionParameters::new(
+            NonZeroU64::new(u64::MAX).unwrap(),
```

Comment:
you can use [`NonZeroU64::MAX`](https://doc.rust-lang.org/std/num/type.NonZeroU64.html#associatedconstant.MAX) instead

Reply:
cool